_chapters/chapter_01.md

```diff
@@ -72,13 +80,21 @@ layout: rails_tutorial
 {% endhighlight %}
 {% endsteps %}
 
+{% aside %}
+  The command "bundle" might sound familiar to you. If you did the install instructions for this tutorial, you probably ran `$gem install bundler`. 
+
+  [Bundler](http://bundler.io/) is a program that helps you manage all of the other little programs that help Rails run and all of the commands related to that program start with `$bundle`. 
+
+  You don't need to worry about this much for the tutorial but if you're interested in it, you can learn more about them on their website. 
+{% endaside %}
```

Comment:
Is this extra info about Bundler necessary? This is the only time Bundler gets used throughout the entire tutorial, and it gets used indirectly.

Bundler is really awesome, but without more context in the tutorial I'm not sure we gain much by adding this little detail.